inst/using-the-dccvalidator-app-amp-ad.Rmd

```diff
@@ -46,25 +52,26 @@ requirements, validating and reviewing the metadata, and uploading the dataset.
 
 ## General Process Overview
 
-1. Contact the AMP-AD team to discuss the study and the expected data. Receive
+1. Contact the Data Liaison (juliane.schneider@sagebase.org) to discuss the study and the expected data. Receive
 staging folder synIDs for each expected dataset.
-2. Upload documentation and validate metadata + manifest files in [dccvalidator](https://shinypro.synapse.org/users/kwoo/dccvalidator-app/).
-3. Contact the AMP-AD team when all files pass validation. The team will verify
+2. Using the templates located at the bottom of the [Onboarding page](https://www.synapse.org/#!Synapse:syn24986618), create a study description, methods description(s), and acknowledgement statement.
+3. Upload the study description, methods description and acknowledgement statement using the [Submission Form](https://www.synapse.org/#!Synapse:syn25051271)
+4. Validate metadata + manifest files in [dccvalidator](https://shinypro.synapse.org/users/kwoo/dccvalidator-app/).
+5. Contact the AMP-AD team when all files pass validation. The team will verify
```

Comment:
Should this have an email address for the AMP-AD team or just leave vague since they _should_ be in contact with someone already?

Reply:
@Aryllen I think it's always better to have more than less info when telling someone to do something, so I'll put that address in.